pkg/commands/workload_apply_test.go

```diff
@@ -3828,6 +3926,1336 @@ Skipping workload %q`,
 				clitesting.ToInteractTerminal("❓ Really update the workload %q? [yN]: m", workloadName),
 				clitesting.ToInteractTerminal("❓ Really update the workload %q? [yN]: n", workloadName), workloadName),
 		},
+		{
+			Name: "update - replace update strategy",
+			Args: []string{flags.FilePathFlagName, "testdata/replace-update-strategy/all-fields-workload.yaml",
+				flags.TypeFlagName, "my-type",
+				flags.UpdateStrategyFlagName, commands.ReplaceUpdateStrategy, flags.YesFlagName},
```

Comment:
pass flag value in string
```suggestion
				flags.UpdateStrategyFlagName, "replace", flags.YesFlagName},
```

Reply:
ok, changed